src/main/java/de/tum/in/www1/artemis/web/rest/ExerciseResource.java

```diff
@@ -169,10 +173,10 @@ else if (authCheckService.isAtLeastTeachingAssistantForExercise(exercise, user))
         }
 
         // TODO CZ: load results of submissions eagerly to prevent additional database calls
-        List<ExampleSubmission> exampleSubmissions = this.exampleSubmissionRepository.findAllByExerciseId(exerciseId);
+        Set<ExampleSubmission> exampleSubmissions = this.exampleSubmissionRepository.findAllWithEagerResultByExerciseId(exerciseId);
```

Comment:
can we remove the above TODO now?

Reply:
Yes, removed the outdated todo in 0e13167fc5c6b10fe64381663e2fc069fcdc5357